keyboards/vitamins_included/vitamins_included.h

```diff
@@ -21,5 +48,4 @@
     )
 
 #define KC_LAYOUT_ortho_4x12 KC_KEYMAP
-
-#endif
+#define LAYOUT_ortho_4x12 KEYMAP
```

Comment:
Adds backwards compatibility for keymaps that use `KEYMAP` as the layout macro name.

```suggestion
#define LAYOUT_ortho_4x12 LAYOUT
#define KEYMAP LAYOUT
```